spellcast.py

```diff
@@ -88,6 +97,19 @@ def __init__(self):
         Reads words from a file and initializes various attributes needed for
         managing the game board.
         """
+        self.words = set()
+        self.letter_values = {}
+        self.letter_multipliers = defaultdict(lambda: 1)
+        self.word_multipliers = defaultdict(lambda: 1)
+        self.board = []
+        self.row_count = 0
+        self.column_count = 0
+        self.total_count = Counter()
+        self.board_value = {}

```

Comment:
It looks like `board_value` is never actually used, it can probably be removed.

Reply:
Removed.